packages/portfolio-deploy/scripts/deploy-cli.ts

```diff
@@ -34,12 +34,16 @@ type ParsedArgs = {
   description?: string;
 };
 
+const errOn = opts => ({
```

Comment:
I'm not sure this indirection is worth it, but if you do want this function please make the name closer to what it's doing: `inheritStdio`

Reply:
rather: `inheritSterr`.